packages/react-components/react-tabs/src/stories/TabListWithPanels.stories.tsx

```diff
@@ -0,0 +1,148 @@
+import { makeStyles, shorthands } from '@griffel/react';
+import * as React from 'react';
+import { SelectTabData, SelectTabEvent, Tab, TabList } from '../index';
+import { AirplaneRegular, AirplaneTakeOffRegular, TimeAndWeatherRegular } from '@fluentui/react-icons';
+import { TabValue } from '../Tab';
+import { tokens } from '@fluentui/react-theme';
+
+const useStyles = makeStyles({
```

Comment:
How about usePanelStyles here? I at first thought you were applying a bunch of styles to your tabs

Reply:
Or Demo styles.....i was also confused by 'root', thinking that was being applied to a slot, and not just some wrapper div.

Reply:
I will go back through all the tab stories to update how I name useStyles.